de.dlr.sc.virsat.model.edit.test/src/de/dlr/sc/virsat/model/dvlm/json/AUuidAdapterTest.java

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ * Copyright (c) 2020 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.dvlm.json;
+
+import static org.junit.Assert.assertThrows;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.impl.ResourceImpl;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import de.dlr.sc.virsat.model.dvlm.types.impl.VirSatUuid;
+
+public abstract class AUuidAdapterTest {
+
+	protected XmlAdapter<?, ?> adapter;
+	protected static final VirSatUuid UUID = new VirSatUuid();
+	protected Resource resourceImpl;
+	protected ResourceSet resourceSet;
+	protected XmlAdapter<?, ?> adapterNoRs;
```

Comment:
Rs NoRs stands for Adapter with ResourceSet and Adapter without?

Reply:
See above